pkg/storage/stores/tsdb/index/index.go

```diff
@@ -61,9 +61,6 @@ const (
 	fingerprintInterval = 1 << 10
 
 	millisecondsInHour = int64(time.Hour / time.Millisecond)
-
-	// The format that will be written by this process
-	LiveFormat = FormatV3
```

Comment:
Could we keep the "live" format for tests? That way, we don't need to update index tests every time a new version is introduced.

Reply:
May be calling it as `latestFormat` instead (private to the package and not exported) may be better I think.

Reply:
Hmm. while implementing it, realized even private `latestFormat` is not needed as all the tests were written to cover both `FormatV2` and `FormatV3` already.

Didn't see the need for it. So removed again. I guess we can add when the needs shows up in future.